install/antennas.csv

```diff
@@ -419,6 +419,7 @@ Trimble Navigation Ltd.,TRM115000.00,1441052893,TGHR,0.002,0,0,0,2019-11-01T00:1
 Trimble Navigation Ltd.,TRM115000.00,1441052895,CHTI,0.002,0,0,0,2019-10-31T00:35:08Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM115000.00,1551016647,TRNG,0.055,0,0,0,2019-10-21T00:48:16Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM115000.00,1551016648,HAST,0.055,0,0,0,2019-11-14T01:14:08Z,9999-01-01T00:00:00Z
+Trimble Navigation Ltd.,TRM115000.00,1551016652,SSFT,0.002,0,0,0,2024-10-16T19:58:38Z,9999-01-01T00:00:00Z
```

Comment:
Hey @EvanCGNS ! Have you checked with the the RIM team if the hight is actually 0.002 m?

Reply:
Yep, confirmed it's 0.002m!